source/Halibut.Tests/Support/LatestClientAndLatestServiceBuilder.cs

```diff
@@ -274,6 +274,7 @@ async Task<IClientAndService> IClientAndServiceBuilder.Build(CancellationToken c
 
         public async Task<ClientAndService> Build(CancellationToken cancellationToken)
         {
+            var logger = new SerilogLoggerBuilder().Build().ForContext<LatestClientAndLatestServiceBuilder>();
```

Comment:
or now, this logger is only needed in the else if (serviceConnectionType == ServiceConnectionType.PollingOverWebSocket) section, right?